src/main/cpp/blaze_util.cc

```diff
@@ -74,6 +76,32 @@ bool GetNullaryOption(const char *arg, const char *key) {
   return true;
 }
 
+std::vector<std::string> SearchNaryOption(const vector<string>& args,
+                              const char *key) {
+  vector<std::string> values;
+  for (vector<string>::size_type i = 0; i < args.size(); ++i) {
+    if (args[i] == "--") {
+      // If the current argument is "--", all following args are target names.
+      // If 'key' was not found, 'value' is nullptr and we can return that.
+      // If 'key' was found exactly once, then 'value' has the value and again
+      // we can return that.
+      // If 'key' was found more than once then we could not have reached this
+      // line, because we would have broken out of the loop when 'key' was found
```

Comment:
where's the break?

Reply:
the `return` line below would be equivalent.

Reply:
removed the comment here, only leaving
```
// If the current argument is "--", all following args are target names.
```
hope the rest should be self explanatory.